cgi/product_multilingual.pl

```diff
@@ -615,6 +615,34 @@ ($$)
 		
 	}
 	
+	
+	# product check
+	
+	if ($admin) {
```

Comment:
What do you think about splitting this up to a new user group? Being able to delete products is potentially much more destructive than the ability to set the checked state.

Reply:
That's right, we can create a moderators group.